ruby/vampires.rb

```diff
@@ -0,0 +1,70 @@
+
+applicant_name = #{name}
```

Comment:
This will error because name is an undefined variable and `#{` is a syntax error